etc/blocks/ui-examples/ui/src/pages/PlAdvancedFilterPage.vue

```diff
@@ -0,0 +1,238 @@
+<script setup lang="ts">
+import type { FilterSpec, ListOptionBase, SUniversalPColumnId } from '@platforma-sdk/model';
+import { PlAdvancedFilter, PlBlockPage, PlCheckbox, PlDropdown } from '@platforma-sdk/ui-vue';
+import { computed, ref, watch } from 'vue';
+
+const uniqueValuesByColumnOrAxisId: Record<string, ListOptionBase<string>[]> = {
+  1: [{ value: '1', label: 'Value 1' }, { value: '2', label: 'Value 2' }],
+  2: [{ value: '3', label: 'Value 3' }, { value: '4', label: 'Value 4' }],
+  3: [{ value: '5', label: 'Value 5' }, { value: '6', label: 'Value 6' }],
+};
+const uniqueValuesByAxisIdx: Record<string, Record<number, ListOptionBase<string>[]>> = {
+  1: { 0: [{ value: 'axisValue1', label: 'Axis Value 1' }, { value: 'axisValue2', label: 'Axis Value 2' }] },
+};
+
+const options = [
+  {
+    id: '1' as SUniversalPColumnId,
+    info: {
+      label: 'Column 1',
+      error: false,
+      axesToBeFixed: [{
+        idx: 0,
+        label: 'Axis 1 label',
+        info: {
+          label: 'Axis 1 label',
```

Comment:
why we need `label` inside `axesToBeFixed[0].label` and inside `axesToBeFixed[0].Info.label`?